plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/ModuleUtils.java

```diff
@@ -53,11 +54,11 @@ public static String getServiceId(IModule module) {
         if (serviceId != null) {
           return serviceId;
         }
-      } catch (CoreException | IOException ex) {
+      } catch (CoreException | IOException | SAXException ex) {
```

Comment:
SAXException should be a separate catch block that doesn't log. A comment at most per #1855